src/router/routes_conf.ts

```diff
@@ -9,6 +9,12 @@ type RoutesConf = {
 };
 
 const routesConf: RoutesConf = {
+  api_login: {
```

Comment:
Could you move this backend REST API endpoint into global app config ride_to_work_by_bike_config.toml, please?

Reply:
API endpoint can be moved to `ride_to_work_by_bike_config.toml`, but when we do that, E2E tests fail, because they directly import `routesConf`, and because the `process.env` variable is not available in Cypress test environment.
It seems we need to either:
write conditional clause for config
```
if (process.env.RIDE_TO_WORK_BY_BIKE_CONFIG) {
  rideToWorkByBikeConfig = JSON.parse(
    process.env.RIDE_TO_WORK_BY_BIKE_CONFIG,
  );
}
```
And add fallback values to `routes_conf.ts` file (for test environment). Or somehow use the `env` parameter in `cypress.config.js` [configuration](https://docs.cypress.io/guides/references/configuration#Global).
I made it work using the first option, but not sure if that is the preferred approach.

Reply:
If you look carefully at the source code of some e2e tests, we use `cy.task('getAppConfig', process)` method to access to global app config.

https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/5013769a671767f21cc0e115d446f8dec885005e/test/cypress/e2e/home.spec.cy.js#L18

Cypress config file with registered `getAppConfig` func as custom component/e2e task.

 https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/5013769a671767f21cc0e115d446f8dec885005e/cypress.config.js#L11-L25

Reply:
Ok, sorry, wrong presumption on my side. Updated.